### DISCUSSION TITLE
🔊 Initiate custom sounds through Companion board?

### DISCUSSION THREAD

**Discussion by croogie** [gcormier/megadesk#161] [Q&A]
Hello floks!

Do you know if there's a way to initiate custom sounds through Companion board? What I have in my mind is that these sounds could remind me to switch position. It can be handled by Home Assistant but I need to initiate sounds somehow.

In the `megadesk_companion.yaml` I see there are several buttons defined. Is there a way to define additional one or two with different sounds played, so I can use them in Home Assistant.

**Comment by gcormier** [comment]
You can look at https://github.com/gcormier/megadesk/blob/master/Serial.md

There are commands to play a tone at a specific freq/duration.

You'd have esphome send those commands.

**Comment by AnAnalogGuy** [comment]
@croogie 
MegaConnect is a small Desktop app working with HA which doesn't remind you to switch position but even better allows you to define patterns of positions (like move to height A, after 10 Minutes, move to height B, .. and so on) and then automatically changes positions following the patterns you have defined. 

I.e. first pattern changes between two positions every 15 minutes, second pattern has 3 different positions that are alternated. The pattern (both positions and duration of each position) can be freely configured.

![screenshot_megaconnect](https://github.com/gcormier/megadesk/assets/70239317/1b2f0014-aceb-41e7-8d01-f2ef10b5a7f0)

https://github.com/gcormier/megaconnect
